src/main/java/sirius/web/dispatch/HelpDispatcher.java

```diff
@@ -124,6 +130,7 @@ private DispatchDecision serveTopic(WebContext ctx, String uri) {
 
     private Template resolveTemplate(String uri) {
         try {
+            uriHandlers.forEach(provider -> provider.handle(uri));
```

Comment:
shouldn't we also supply the uri handler with a signal "template exists" (?)

Reply:
added =)